config/main.py

```diff
@@ -1983,8 +1983,23 @@ def load_minigraph(db, no_service_restart, traffic_shift_away, override_config,
                         fg='magenta')
             raise click.Abort()
 
-        # Dependency check golden config json
+        cfgdb_clients = db.cfgdb_clients
+        # Pick the host config in case of multi-asic
+        config_db = cfgdb_clients[DEFAULT_NAMESPACE]
+
+        cm = None
+        try:
+            cm = ConfigMgmt(configdb=config_db)
+            cm.validateConfigData()
+        except Exception as ex:
```

Comment:
>except [](http://example.com/codeflow?start=8&length=6)

I see there are several options or design decisions. How about we move the options into a conf file, and then we can provide a default one. The benefit is possibility of runtime override if any urgency happen, or any potential phased rollout (even we do not plan for it).
1. yang_invalid_final_config_throw: yes/no
2. yang_invalid_final_config_logging: yes/no
3. yang_invalid_final_config_loglevel: WARN/ERROR/etc
4. namespaces: default/all
5. etc

Reply:
I think we want to block the process since currently Golden Config is already yang validated based on recent deployement. If it fail, there must be some modification or corner case. And we can use this failure to push the fix from feature owner. 
If we let it pass, we will leave bad config into PROD, which will take long time fix. Is that what we want?